react/components/PaginationDots.tsx

```diff
@@ -56,19 +53,12 @@ const PaginationDots: FC<Props> = ({ controls, totalItems }) => {
   )
 
   const handleDotClick = (index: number) => {
-    const slideToGo = passVisibleSlides ? index * slidesPerPage : index
-    const isLastDot = index === slideIndexes.length - 1
-    const isExactDivision = totalItems % slidesPerPage === 0
-    const overslideThreshold =
-      !isExactDivision && isLastDot ? Math.floor(totalItems / slidesPerPage) : 0
+    const pageDelta =
```

Comment:
Could you add a comment here explaining this pageDelta logic?

Reply:
Sure thing!